### DISCUSSION TITLE
Hydra not able to connect postgres 11.6

### DISCUSSION THREAD

**Discussion by abhishakeonteddu** [ory/hydra#3437] [Q&A] [answered]
Hello I am using hydra 2.5.3 and postgres 11.6 when i deploy using helm chart. Hydra pod is giving below error. any clue.

time=2023-02-13T16:41:09Z level=info msg=No tracer configured - skipping tracing setup audience=application service_name=Ory Hydra service_version=v2.0.3
time=2023-02-13T16:41:09Z level=warning msg=Unable to determine network, retrying. audience=application error=map[debug: message:Unable to locate the table reason: status:Internal Server Error status_code:500] service_name=Ory Hydra service_version=v2.0.3
time=2023-02-13T16:41:09Z level=error msg=Unable to create service registry. audience=application error=map[debug: message:Unable to locate the table reason: status:Internal Server Error status_code:500] service_name=Ory Hydra service_version=v2.0.3
Error: Unable to locate the table
### **Unable to locate the table**
Usage:
  hydra serve all [flags]

Flags:
  -h, --help   help for all

Global Flags:
  -c, --config strings   Path to one or more .json, .yaml, .yml, .toml config files. Values are loaded in the order provided, meaning that the last config file overwrites values from the previous config file.
      --dev              Disables critical security checks to improve local development experience. Do not use in production.
      --sqa-opt-out      Disable anonymized telemetry reports - for more information please visit https://www.ory.sh/docs/ecosystem/sqa

**Comment by abhishakeonteddu** [comment]
could some one suggest me where is the issue

**Reply by Newbie112222** [reply]
@abhishakeonteddu Did you ever find a solution to this problem?

**Comment by vinckr** [comment]
Hello @abhishakeonteddu 

It is hard for us to know what exactly the issue is without having access to your infra / a reproducible test case. 

From the errors you posted it looks like there is an issue with Kubernetes / helm:

```
Unable to determine network
Unable to create service registry
Unable to locate the table
```

Those are not errors from Ory Hydra but rather from your clusters. 
Maybe you can also ask the k8s community for help, as we are not experts in that. 

Btw. if you dont want to deal with setting up Kubernetes etc. check out [Ory Network](https://console.ory.sh/).

**Reply by krtk6160** [reply, marked as answer]
That's not a problem with kubernetes. I had the same problem and enabling automigration fixed it. The `unable to determine network` is misleading as it could have you believe it's a networking problem, even though the networking between PG and hydra was setup correctly.

**Reply by vinckr** [reply]
Oh gotcha! Thanks for sharing your fix.
FYI @Newbie112222

**Reply by vc-lijup-nandana** [reply]
Hi krtk6160,
What you have enabled automigration and to what version? We are also facing the same issue in our aks

**Comment by Tlmonko** [comment]
You should enable the automigration

If you use helm chart, here is the option, you should set

```
hydra.automigration.enabled: true
```

**Reply by johnitvn** [reply]
This comment did saved me
